src/table/body-cell/styles.scss

```diff
@@ -418,7 +411,8 @@ $cell-negative-space-vertical: 2px;
       }
       @include cell-padding-inline-start($editing-cell-padding-inline);
       @include cell-padding-inline-end($editing-cell-padding-inline);
-      @include cell-padding-block($editing-cell-padding-block);
+      @include cell-padding-block-start($editing-cell-padding-block);
+      @include cell-padding-block-end(calc($editing-cell-padding-block + 1px));
```

Comment:
Adding a small extra space here ensures better input alignment especially in compact mode.